Extension/src/LanguageServer/client.ts

```diff
@@ -745,6 +751,43 @@ class DefaultClient implements Client {
         }
     }
 
+    private promptCompileCommands(params: CompileCommandsPaths) : void {
+        if (this.configuration.Configurations[this.configuration.CurrentConfiguration].compileCommands !== undefined) {
+            return;
+        }
+
+        let showCompileCommandsSelection: PersistentState<boolean> = new PersistentState<boolean>("CPP.showPromptCompileCommands", true);
+        if (!showCompileCommandsSelection.Value) {
+           return;
```

Comment:
>    [](start = 8, length = 3)

nit: looks like indentation is off by one space here

Reply:
Fixed!